modal-js/src/app.ts

```diff
@@ -61,8 +66,45 @@ export type SandboxCreateOptions = {
 
   /** List of ports to tunnel into the sandbox without encryption. */
   unencryptedPorts?: number[];
+
+  /** Whether to block all network access from the sandbox. */
+  blockNetwork?: boolean;
+
+  /** List of CIDRs the sandbox is allowed to access. If None, all CIDRs are allowed. Cannot be used with blockNetwork. */
+  cidrAllowlist?: string[];
```

Comment:
We added this one in Python after `block_network` but in retrospect I wonder if a single parameter where `cidr_allowlist=[]` to block all network access would have been preferable.

OTOH maybe that's tricky to represent in the protobuf layer.